src/lib/y2network/config_writer/sysconfig_interfaces.rb

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+require "yast"
+
+module Y2Network
+  module ConfigWriter
+    class SysconfigInterfaces
+      COMMON_KEYS = [
```

Comment:
what is goal of this constant? No documentation and no usage.

Reply:
You are right, that class is basically a placeholder. I am removing it by now.